beacon-chain/core/blocks/block_operations.go

```diff
@@ -742,7 +777,9 @@ func ProcessAttestationNoVerify(
 func VerifyIndexedAttestation(ctx context.Context, beaconState *stateTrie.BeaconState, indexedAtt *ethpb.IndexedAttestation) error {
 	ctx, span := trace.StartSpan(ctx, "core.VerifyIndexedAttestation")
 	defer span.End()
-
+	if indexedAtt == nil || indexedAtt.Data == nil || indexedAtt.Data.Target == nil {
+		return fmt.Errorf("nil or missing indexed attestation data: %v", indexedAtt)
```

Comment:
why are we logging the indexedAtt if it's nil or missing?

Reply:
duplicate